src/main/java/com/adobe/target/edge/client/service/DefaultTargetService.java

```diff
@@ -41,6 +42,7 @@ public class DefaultTargetService implements TargetService {
   public static final String SDK_VERSION_KEY = "X-EXC-SDK-Version";
   public static final String SESSION_ID = "sessionId";
   public static final String ORGANIZATION_ID = "imsOrgId";
+  public final String SDK_VERSION;
```

Comment:
Let's make this `private final String sdkVersion;` since it's no longer a constant value.